packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -372,6 +397,7 @@ export default class UserStorageController extends BaseController<
     state?: UserStorageControllerState;
     env?: {
       isAccountSyncingEnabled?: boolean;
+      isNetworkSyncingEnabled?: boolean;
```

Comment:
Like this pattern similar to account syncing. Allows us to control, at a platform level, if these syncing features should be enabled.